src/components/cart/proceed-payment/index.tsx

```diff
@@ -0,0 +1,242 @@
+                <InputField
+                  name="Add your promo code"
+                  isPassword={false}
+                  style={{ width: "238px", height: "33px" }}
+                />
+                <AlertText content={data.offersRadio.alertsText} />
+              </UserCodeSection>
+            ) : null}
+            <Wrapper style={{ marginTop: "24px" }}>
+              <RadioInput
+                type="radio"
+                id="bestOffer"
+                name="referal"
+                value="bestOffers"
+                onClick={handleRadioClick}
+                checked={radioButtons.bestOffers}
+              />
+              <OptionName htmlFor="best-offer">
+                {data.offersRadio.options[1]}
+              </OptionName>
+            </Wrapper>
+          </PaymentContainer>
+        </CartSection>
+      </ContentContainer>
+    </PageSection>
+  );
+};
+export default ProceedPayment;
+
+const PageSection = styled.div`
+  width: 100wh;
+  height: 842px;
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 40px;
+  padding-bottom: 72px;
+`;
+
+const ContentContainer = styled.div`
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  justify-content: space-between;
+  gap: 39px;
+  margin-left: auto;
+  margin-right: auto;
+`;
+
+const ProgressSection = styled.div`
+  height: 55px;
+  width: 66%;
+`;
+
+const CookingInstructionTitle = styled.div`
+  height: 17px;
+  width: 122.95px;
+  color: ${colors.grey_858585};
+  font-family: ${fontFamilies.fontFamilyOsRegular};
+  font-size: ${sizes.size12};
+  letter-spacing: ${sizes.sizeNeg0_24};
+  line-height: ${sizes.size17};
+  margin-left: 21px;
+  margin-top: 17px;
+  margin-bottom: 15px;
+`;
+
+const CartSection = styled.div`
+  width: 100%;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  gap: 23px;
+  box-sizing: border-box;
+`;
+
+const StepsSection = styled.div`
+  width: 634px;
+`;
+
+const StepTitle = styled(Label)`
+  height: "30px";
+  color: ${colors.black_000000};
+  fontfamily: ${fontFamilies.fontFamilyOsSemiBold};
+  fontsize: ${sizes.size22};
+  letterspacing: ${letterSpacing.space0};
+  lineheight: ${sizes.size30};
+`;
+
+const PaymentContainer = styled.div`
+  width: 305px;
+  height: 480px;
+  margin-top: 74px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  padding: 15px 17px 12px 18px;
+  border-radius: 6px;
+  box-shadow: 0 2px 10px 0 ${colors.black_000000_1};
+`;
+
+const RadioTitle = styled.div`
+  height: 23px;
+  color: ${colors.black_000000};
+  font-family: ${fontFamilies.fontFamilyOsRegular};
+  font-size: ${sizes.size17};
+  letter-spacing: ${letterSpacing.spaceNeg0_41};
+  line-height: ${sizes.size23};
+  margin-bottom: 17px;
+`;
+
+const Wrapper = styled.div`
+  display: block;
+  height: 16px;
+  top: 0;
+`;
+
+const RadioInput = styled.input`
```

Comment:
Create common component for RadioInput

Reply:
Radio input now derived from common component InputField and styled